components/tag-list/demo/tag-list.html

```diff
@@ -71,8 +71,8 @@ <h2>Clearable Tag List</h2>
 					import { getNextFocusable } from '../../../helpers/focus.js';
 
 					document.addEventListener('d2l-tag-list-item-clear', (e) => {
+						console.log(`d2l-tag-list-item-clear event dispatched. Value: ${e.target.text}`);
```

Comment:
Should we add keys to the demo, as a good example? And then we can add it hear alongside the value, to show how to get both